### PR TITLE
fetch server names from dbcopy api

### PR DESCRIPTION
**Pull request by vinay-ebi**
Fetch server names from dbcopy api , removed json file dependency

### CODE REVIEW

File: src/ensembl/production/datacheck/config.py [hunk 1]
```diff
@@ -38,6 +41,29 @@ def get_app_version():
             version = f.read()
     return version
 
+def get_server_names(url, flag=0):
```
Comment (marcoooo) [inline src/ensembl/production/datacheck/config.py:44]: There is this nice core thing I created for loading remote configs: https://github.com/Ensembl/ensembl-py/blob/d8859c0215b641cb6db4b4e8ac7037ffcc167a76/src/python/ensembl/utils/rloader.py 
I guess we could reuse may be?

**Review by marcoooo — CHANGES_REQUESTED** [review]

File: src/ensembl/production/datacheck/config.py (hunk 1 above)
Comment (marcoooo) [reply src/ensembl/production/datacheck/config.py:44]: BTW, the fall back could just be an empty list. So the app start, but empty.

Comment (vinay-ebi) [reply src/ensembl/production/datacheck/config.py:44]: we can reuse the RemoteFileLoader

File: src/ensembl/production/datacheck/config.py [hunk 2]
```diff
@@ -109,8 +127,12 @@ class DatacheckConfig(EnsemblConfig):
     ES_HOST = os.environ.get('ES_HOST', EnsemblConfig.file_config.get('es_host', 'localhost'))
 
     ES_PORT = os.environ.get('ES_PORT', EnsemblConfig.file_config.get('es_port', '9200'))
+    
+    ES_INDEX = os.environ.get('ES_INDEX', EnsemblConfig.file_config.get('es_index', f"datacheck_results_{EnsemblConfig.ENS_VERSION}"))
+    
+    GET_SERVER_NAMES = os.environ.get('GET_SERVER_NAMES', EnsemblConfig.file_config.get('get_server_ names', 0))
```
Comment (marcoooo) [inline src/ensembl/production/datacheck/config.py:133]: Probably a typo here, the key searched has a space within.

**Review by marcoooo — CHANGES_REQUESTED** [review]

**Review by marcoooo — APPROVED** [review]

**Pull request merged**